doc/mgr/influx.rst

```diff
@@ -55,6 +54,15 @@ Additional optional configuration settings are:
 :port: InfluxDB server port.  Default 8086
 :ssl: Use https connection for InfluxDB server. Use "true" or "false". Default false
 :verify_ssl: Verify https cert for InfluxDB server. Use "true" or "false". Default true
+:destinations: Set multiple influxdb destinations.   Set as a JSON array of objects.  The property 'hostname' is required in each object.  Other properties will be taken from the global config-key settings if they are not set in the objects.
```

Comment:
It would be better to have add/rm/ls commands for managing this -- expecting users to type out valid JSON is usually a risky proposition.